### ISSUE TITLE
ReduceLROnPlateauScheduler bug on updating val loss

### ISSUE THREAD

**Issue by martinBmeza** [sooftware/pytorch-lr-scheduler#4]
In the ReduceLROnPlateauScheduler, the code is updating val_loss attribute when the current loss doesnt improve. This may be an unwanted feature, as the val_loss could be oscillating, and not allowing the scheduler to decrease the learning rate to improve the val_loss. 

precisely, line 59 in reduce_lr_on_plateau_lr_scheduler.py must be deleted.
